packages/loader/container-loader/src/utils.ts

```diff
@@ -78,18 +78,20 @@ export interface IParsedUrl {
  * Utility api to parse the IResolvedUrl.url into specific parts like querystring, path to get
  * deep link info etc.
  * Warning - This function may not be compatible with any Url Resolver's resolved url. It works
- * with urls of type: protocol://<string>/.../..?<querystring>
+ * with urls of type: [http | https]://<host>/<tenantId>/<containerId>/<stringPath>/.../..?<querystring>
  * @param url - This is the IResolvedUrl.url part of the resolved url.
  * @returns The IParsedUrl representing the input URL, or undefined if the format was not supported
  * @legacy
  * @alpha
  */
 export function tryParseCompatibleResolvedUrl(url: string): IParsedUrl | undefined {
 	const parsed = new URL(url);
-	if (typeof parsed.pathname !== "string") {
-		throw new LoggingError("Failed to parse pathname");
+	if (parsed.protocol !== "http:" && parsed.protocol !== "https:") {
+		throw new LoggingError("Failed to parse url, invalid protocol");
```

Comment:
I want to suggest this, but the URL might technically be user-content (or similar classification)? Which means we'd need to tag it appropriately if/when we log it to point that out (assuming this error ever gets logged by us, which might be a hard question to answer with certainty...?).
```suggestion
		throw new LoggingError(`Failed to parse url, invalid protocol '${parsed.protocol}'`);
```

Reply:
I considered this, but decided to omit it both for the hypothetical user-data risk but also because we probably kind of know already what it's going to be if it's wrong.  `fluid-odsp` is the only one we'd have any real chance of seeing in the wild, and we'd have other surrounding telemetry about the driver used that could help confirm.